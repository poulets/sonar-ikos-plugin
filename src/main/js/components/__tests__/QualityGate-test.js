import React from 'react';
import {shallow} from 'enzyme';
import QualityGate from '../QualityGate';

it ('QualityGate-OK', () => {
  expect(
    shallow(
      <QualityGate
       qg_status={'OK'}
      />

    )
  ).toMatchSnapshot();
});

it ('QualityGate-WARNING', () => {
  expect(
    shallow(
      <QualityGate
       qg_status={'WARN'}
      />

    )
  ).toMatchSnapshot();
});

it ('QualityGate-KO', () => {
  expect(
    shallow(
      <QualityGate
       qg_status={'KO'}
      />

    )
  ).toMatchSnapshot();
});
