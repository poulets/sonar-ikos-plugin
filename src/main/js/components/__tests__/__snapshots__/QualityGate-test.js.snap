exports[`test QualityGate-KO 1`] = `
<span
  className="level level-KO custom-abc">
  Failed
</span>
`;

exports[`test QualityGate-OK 1`] = `
<span
  className="level level-OK">
  Passed
</span>
`;

exports[`test QualityGate-WARNING 1`] = `
<span
  className="level level-WARN">
  Warning
</span>
`;
